.github/workflows/auto-publish.yml

```
@@ -0,0 +1,21 @@
+# Example: Override respecConfig for W3C deployment and validators.
+name: Echidna Auto-publish
+on:
+  pull_request: {}
+  push:
+    branches: [main]
```

This will regenerate the publication on _any_ change in the repository (e.g., any change in the test directory). We may not want to do this. By changing these lines, we may avoid this:
```suggestion
on:
  pull_request:
    paths: ["spec/**"]
  push:
    branches: [main]
    paths: ["spec/**"]
```

@msporny may also want to review this to check if I am right; I am not a very good gh action expert.